ote/src/cljs/ote/app/controller/transit_visualization.cljs

```diff
@@ -87,7 +87,7 @@
   "Sort route changes according to change date and route-long-name: Earliest first and missing date last."
   [show-no-change changes]
   (let [;; Removed in past routes won't be displayed at the moment. They are ended routes and we do not need to list them.
-        removed-in-past (sort-by (juxt :route-long-name :route-short-name) (filterv #(and (= :removed (:change-type %)) (nil? (:change-date %))) changes))
+        ;removed-in-past (sort-by (juxt :route-long-name :route-short-name) (filterv #(and (= :removed (:change-type %)) (nil? (:change-date %))) changes))
```

Comment:
Is it necessary to save this commented out code?

Reply:
Very good question, previously it was _unused_ and here it's just commented. Previous comment explains why, but I didn't dare to remove it as part of this yet.